tests/mock_eposerver.py

```diff
@@ -0,0 +1,28 @@
+from dxlclient.service import ServiceRegistrationInfo
+from tests.mock_requesthandlers import *
+
+class MockEpoServer(object):
+    def __init__(self, client, id_number=0):
+        self._client = client
+        self.id_number = id_number
+
+        # Create DXL Service Registration object
+        self._service_registration_info = ServiceRegistrationInfo(
+            self._client,
+            "/mcafee/service/epo/remote",
+        )
+        self._service_registration_info._ttl_lower_limit = 5
```

Comment:
So that in the case of local runs that are aborted or crash before unregistering the service, the registration does not hang around in the DXL Broker's service registry for too long.

It's relevant for the OpenDXL ePO Client because when the client is created it grabs all of the currently registered services with the prefix "/mcafee/service/epo/remote", meaning that any that share a registration for the same name of an ePO Server (e.g. "localTest") will cause problems with the local differentiation when the client tries to send a remote command to the correct DXL Service.